v0/destinations/vero/data/VeroTrackConfig.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "sourceKeys": ["userId", "context.traits.email", "traits.email", "anonymousId"],
```

Comment:
Need to fix as per other mappings (identify) also we should add support for `identity.email`

Reply:
`identity.id` is fixed. Didn't add `identity.email` as it was not working as expected. If we're sending only `identity.email` without `identity.id`, Vero is checking for a user with id as `identity.email` (not email as `identity.email`) and creating a user with id as `identity.email` even though there exists a user with same email.

Reply:
Now as we decided we'll send `Id` in all cases (either userId or anonId), Vero will use `Id` for identifying the user but also will update the  email if `identity.email` is different from what is already set. So will add `identity.email` field.

Reply:
Make sense (but that would not be a proper case of updating `email` attribute for a user, identify should be used to update email)
What if in the track call the same user has id as `anonymousId` in vero and his email is `emailA` if in track id is `userId` and emails is same as `emailA` will his id will get updated?
@sandeepdsvs ?

Reply:
No @utsabc, it creates a new user with id as `userId`

Reply:
Got it